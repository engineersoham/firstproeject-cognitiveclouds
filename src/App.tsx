import React from 'react';
import logo from './logo.svg';
import './App.css';
import { Route, Routes } from 'react-router-dom';
import Nav from './component/Nav';
import Countrydetails from './component/Countrydetails';
import Weatherdetails from './component/Weatherdetails';

function App() {
  return (
    <>
    <Nav/>
    <Routes>
      <Route path='/country/:country' element={<Countrydetails/>}/>
      <Route path='/weather/:capital' element={<Weatherdetails/>}/>
    </Routes>
    </>
  );
}

export default App;