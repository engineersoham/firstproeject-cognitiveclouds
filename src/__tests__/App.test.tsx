import { screen,render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import App from "../App";

it('app snap',()=>{
    const tree = render(
        <BrowserRouter><App/></BrowserRouter>
    )
    expect(tree).toMatchSnapshot()
    screen.debug()
})