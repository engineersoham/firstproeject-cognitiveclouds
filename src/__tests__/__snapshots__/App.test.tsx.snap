// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app snap App 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div>
          <header
            class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
          >
            <div
              class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-1t29gy6-MuiToolbar-root"
            >
              <h5
                class="MuiTypography-root MuiTypography-h5 css-1f8a19r-MuiTypography-root"
              >
                Weather Report
              </h5>
              <form
                action=""
              >
                <input
                  id="browsers"
                  name="name"
                  placeholder="enter country name"
                  style="height: 2rem; margin: 10px;"
                  type="text"
                  value=""
                />
                <button
                  disabled=""
                  style="height: 2.4rem; color: black; margin-top: 5px;"
                  type="submit"
                >
                  Search
                </button>
              </form>
            </div>
          </header>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div>
        <header
          class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
        >
          <div
            class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-1t29gy6-MuiToolbar-root"
          >
            <h5
              class="MuiTypography-root MuiTypography-h5 css-1f8a19r-MuiTypography-root"
            >
              Weather Report
            </h5>
            <form
              action=""
            >
              <input
                id="browsers"
                name="name"
                placeholder="enter country name"
                style="height: 2rem; margin: 10px;"
                type="text"
                value=""
              />
              <button
                disabled=""
                style="height: 2.4rem; color: black; margin-top: 5px;"
                type="submit"
              >
                Search
              </button>
            </form>
          </div>
        </header>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
